Source/WebKit/UIProcess/API/glib/WebKitFeature.h.in

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2023 Igalia S.L.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public License
+ * along with this library; see the file COPYING.LIB.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA.
+ */
+
+@API_SINGLE_HEADER_GUARD@
+
+#ifndef WebKitFeature_h
+#define WebKitFeature_h
+
+#include <glib-object.h>
+#include <@API_INCLUDE_PREFIX@/WebKitDefines.h>
+
+G_BEGIN_DECLS
+
+/**
+ * WebKitFeatureStatus:
+ * @WEBKIT_FEATURE_STATUS_EMBEDDER: Feature that adjust behaviour for
+ *   specific application needs. The feature is not part of a Web platform
+ *   feature, not a mature feature intended to be always on.
+ * @WEBKIT_FEATURE_STATUS_UNSTABLE: Feature in development. The feature
+ *   may be unfinished, and there are no guarantees about its safety and
+ *   stability.
+ * @WEBKIT_FEATURE_STATUS_INTERNAL: Feature for debugging the WebKit engine.
+ *   The feature is not generally useful for user or web developers, and
+ *   always disabled by default.
+ * @WEBKIT_FEATURE_STATUS_DEVELOPER: Feature for web developers. The feature
+ *   is not generally useful for end users, and always disabled by default.
```

Comment:
So these are _web developer_ features, not WebKit developer features? It's news to me. :P 

What's the point of having features for developers that are always disabled? The only way to turn them on is to use Safari Tech Preview, correct? Or does the stable Safari allow toggling individual features?

Reply:
Safari stable has a hidden “Develop” menuy which can be enabled, and features listed there individually. IIUC in Safari TP the menu is always shown.